app/templates/carpools/error.html

```diff
@@ -0,0 +1,16 @@
+{%- extends "_template.html" %}
+
+{% block site %}
+<div class="content carpool-error">
+
+    <div class="fullscreen">
+        {% include '_flash_messages.html' %}
+        <div class="alert alert-danger">
+            Sorry, you can be in at most ten carpools.
```

Comment:
Could we use a flashed message here (i.e. call `flash("Sorry, you can be ...")` in `carpools/views.py` and then render the template) so that the `error.html` template is generic and could be used to show any error message related to carpools?